W-16622164-configure-pce4-studio-kt (#575)
mulesoft/docs-studio: modules/ROOT/pages/pce-configuration.adoc

include::_attributes.adoc[]
endif::[]

> The following topics cover how to configure Anypoint Private Cloud Edition (Anypoint Platform PCE) in Anypoint Studio:

Reviewer: ```suggestion
To configure Anypoint Private Cloud Edition (Anypoint Platform PCE) in Anypoint Studio, follow the instructions for the version you are installing:
```
